rotkehlchen/tests/api/blockchain/test_substrate.py

```diff
@@ -51,7 +51,7 @@ def test_add_ksm_blockchain_account_invalid(rotkehlchen_api_server):
 @pytest.mark.vcr()
 @pytest.mark.parametrize('number_of_eth_accounts', [0])
 @pytest.mark.parametrize('kusama_manager_connect_at_start', [(KusamaNodeName.OWN,)])
-@pytest.mark.parametrize('ksm_rpc_endpoint', [KUSAMA_TEST_RPC_ENDPOINT])
+@pytest.mark.parametrize('ksm_rpc_endpoint', [KUSAMA_TEST_RPC_ENDPOINT], ids=['KUSAMA_TEST_RPC_ENDPOINT'])  # setting ids to rename the argument to be processed by vcr # noqa: E501
```

Comment:
1.This should have been split in 2 different commits. I said that during the standup. Why was it not done?
2. What does this do? I don't understand. How does this solve the problem? This is not explained at all here or in the other comments. Also don't just copy past the comment. Just explain that this affects all similar fixtures in the test file

Reply:
how it solves the problem, it changes the cassette name, I added more context here:
https://discord.com/channels/657906918408585217/1080783409640648765/1169173684305539134

Reply:
it's in split commits now